build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -318,13 +318,6 @@ public static void PublishSharedFramework(BuildTargetContext c, string outputDir
                 Path.Combine(Dirs.CorehostLatest, HostPolicyBaseName),
                 Path.Combine(SharedFrameworkNameAndVersionRoot, HostPolicyBaseName), true);
 
-            if (File.Exists(Path.Combine(SharedFrameworkNameAndVersionRoot, "mscorlib.ni.dll")))
```

Comment:
> NI contains the IL

I thought only R2R contained them. Thanks for clarifying. I'm leaving the error handling for mscorlib to CoreCLR since apps can choose to delete one or the other.